lib/templates/tabMenuLink.mustache

```diff
@@ -0,0 +1,8 @@
+<li class="res-tabmenu-button">
+	<a class="{{ className }}" {{#title}}title="{{ title }}"{{/title}}>
+	<label>
+		<input type="checkbox" {{checked}}>
+		{{text}}
```

Comment:
The newlines here add extra space, might as well put everything inside the `<label>` on one line (could use CSS of course but that's probably overkill)

![image](https://cloud.githubusercontent.com/assets/7673145/15622805/bde83efc-243a-11e6-9f56-2660333f5100.png)